lib/taskana-core/src/main/java/pro/taskana/impl/TaskServiceImpl.java

```diff
@@ -202,10 +201,10 @@ public Task getTask(String id) throws TaskNotFoundException {
             taskanaEngineImpl.openConnection();
             result = taskMapper.findById(id);
             if (result != null) {
-                setAttachments(result);
-                Classification classification;
+                organizeAttachments(result);
                 try {
-                    classification = this.classificationService.getClassificationByTask(result);
+                    Classification classification = this.classificationService.getClassificationByTask(result);
```

Comment:
können wir da nicht eine andere Lösung finden? Diese Methode finde ich immer noch gewöhnungsbedürftig...

Reply:
Das geht aktuell wegen der auftretenden Exceptions leider nicht. Wir haben eigene Queries für Classification und Attachment. Wir könnten darüber reden exta Queries in eigene Methoden zu verpacken - so bleibt das Exception-Gewirr in den Haupt-Methoden etwas klarer.
Da dies jedoch überall so ist, aktuell, sollte dies ein eigenes Ticket sein - was meinst du @holgerhagen ?